docs/usage.md

```diff
@@ -23,6 +23,16 @@ CodeChecker log -b "make" -o compilation.json
 ```
 cat ./compilation.json
 ```
+
+**What to do if the `compilation.json` is empty?**
+* Make sure that your build system actually invoked the compiler (e.g. `gcc`,`g++`). 
+  In case your software was built once (and the binaries are already generated),
+  the compiler will not be invoked. In this case do a build cleanup (e.g. `make clean`) and 
+  retry to log your build.
+  
+* Make sure that the `CC_LOGGER_GCC_LIKE` environment variable is set correctly and contains your compilers. 
+  For detailed description see the [user guide](user_guide.md#1-log-mode).
```

Comment:
It would be better if documentation used absolute links: `[user guide](/docs/user_guide.md#1-log-mode)`, we use such crossreferences in other places too.